manifests/vhost.pp

```diff
@@ -1188,19 +1188,22 @@
 #   apache::vhost { 'wsgi.example.com':
 #     port                        => 80,
 #     docroot                     => '/var/www/pythonapp',
-#     wsgi_daemon_process         => 'wsgi',
-#     wsgi_daemon_process_options =>
-#       { processes    => 2,
-#         threads      => 15,
+#     wsgi_daemon_process         => {
+#       'wsgi' => {
+#         processes    => '2',
+#         threads      => '15',
```

Comment:
Do you know if we can use integers?  If so, this may be preferable for consistency (even if there is no actual check of the data type).